frontend/src/scenes/dashboard/DashboardHeader.tsx

```diff
@@ -120,19 +120,33 @@ export function DashboardHeader(): JSX.Element | null {
             <PageHeader
                 buttons={
                     dashboardMode === DashboardMode.Edit ? (
-                        <LemonButton
-                            data-attr="dashboard-edit-mode-save"
-                            type="primary"
-                            onClick={() => setDashboardMode(null, DashboardEventSource.DashboardHeader)}
-                            tabIndex={10}
-                            disabled={dashboardLoading}
-                        >
-                            Done editing
-                        </LemonButton>
+                        <>
+                            <LemonButton
+                                data-attr="dashboard-edit-mode-discard"
+                                type="secondary"
+                                onClick={() =>
+                                    setDashboardMode(null, DashboardEventSource.DashboardHeaderDiscardChanges)
+                                }
+                                tabIndex={9}
+                            >
+                                Discard Changes
+                            </LemonButton>
+                            <LemonButton
+                                data-attr="dashboard-edit-mode-save"
+                                type="primary"
+                                onClick={() =>
+                                    setDashboardMode(null, DashboardEventSource.DashboardHeaderSaveDashboard)
+                                }
+                                tabIndex={10}
+                                disabled={dashboardLoading}
+                            >
+                                Save Dashboard
```

Comment:
We use sentence case for buttons, and not title case. [This style guide](https://posthog.com/handbook/growth/marketing/posthog-style-guide) basically applies to all text at PostHog, including in the app.

However in this case we should also follow existing UI conventions – e.g. in insight edit mode these buttons are "Cancel" and "Save". Let's follow that for consistency:
```suggestion
                                Cancel
                            </LemonButton>
                            <LemonButton
                                data-attr="dashboard-edit-mode-save"
                                type="primary"
                                onClick={() =>
                                    setDashboardMode(null, DashboardEventSource.DashboardHeaderSaveDashboard)
                                }
                                tabIndex={10}
                                disabled={dashboardLoading}
                            >
                                Save
```

Reply:
Thanks for pointing this out and updating it! Will keep that in mind. I had tried to match the edit mode wording, but matching the insight page makes a lot more sense.